js/start.js

```diff
@@ -85,27 +85,23 @@ if (process.platform === 'darwin') {
 
 $html.addClass(platformClass);
 
-//put language in the window so all templates and models can reach it. It's especially important in formatting currency.
-//retrieve the stored value, since user is a blank model at this point
-window.lang = localStorage.getItem('lang') || "en-US";
+validateLanguage = function(lang){
+  //check to see if the language exists in the language model
+  if (__.where(languages.get('languages'), {langCode: lang}).length) {
+    return lang;
+  }
+  return "en-US"; //if language is not found, use English
+};
 
 //put polyglot in the window so all templates can reach it
-window.polyglot = new Polyglot({locale: window.lang});
-
-(extendPolyglot = function() {
-  // Make sure the language exists in the languages model
-  if (__.where(languages.get('languages'), {langCode: window.lang}).length) {
-    var language = require('./languages/' + window.lang + '.json');
+window.polyglot = new Polyglot();
 
-    window.polyglot.extend(language);
-  }
-})(window.lang);
-
-updatePolyglot = function(lang){
-  window.lang = lang;
-  extendPolyglot(lang);
+(updatePolyglot = function(lang){
+  lang = validateLanguage(lang);//make sure the language is valid
+  window.lang = lang; //put language in the window so all templates and models can reach it. It's especially important in formatting currency.
   localStorage.setItem('lang', lang);
-};
+  window.polyglot.extend(require('./languages/' + lang + '.json'));
+})(validateLanguage(localStorage.getItem('lang')));
```

Comment:
I don't think we need to call validateLanguage on the arg here because it's called in the first line of the function.